README.md

```diff
@@ -39,6 +39,11 @@ At a minimum, you need to specify `host`, `user`, `password`, `schema` (database
 | 1.0.0.x           | ❌           | ✅          | ✅          | ✅          | ❌          |
 |1.1.0.x            | ❌           | ✅          | ✅          | ✅          | ✅          |
 
+##  dbt dependent packages version compatibility
```

Comment:
don't we need a row in the python compatibility matrix for 1.2.0.x? Maybe it will be introduced with a different PR?

Reply:
Yes Adam, we have added the python compatibility for 1.2.0.x in the other PR.